package.json

```diff
@@ -10,7 +10,9 @@
     "test": "mocha test/*.coffee",
     "test-cov": "cross-env COFFEECOV_INIT_ALL=false mocha --require coffee-coverage/register-istanbul test/*.coffee && istanbul report",
     "it": "mocha it/*.coffee",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post",
+    "license-violations": "node-license-validator -c node-license-violations.json",
+    "license-notice-warns": "node-license-validator -c node-license-notice-warns.json || true"
```

Comment:
Looking at https://github.com/myndzi/node-license-validator/blob/master/cli.js -c seems to be undocumented and I'm not sure it actually works as there is no reference to argv.c in the source.  Seems like the right way to do what you're attempting is to have a single list of licenses and run that twice, first with --warn and second without.